packages/kbn-test/src/functional_tests/start_servers/start_servers.ts

```diff
@@ -40,7 +40,15 @@ export async function startServers(log: ToolingLog, options: StartServerOptions)
       procs,
       config,
       installDir: options.installDir,
-      extraKbnOpts: options.installDir ? [] : ['--dev', '--no-dev-config', '--no-dev-credentials'],
+      extraKbnOpts: options.installDir
+        ? []
+        : [
+            '--dev',
+            '--no-dev-config',
+            '--no-dev-credentials',
+            // Simulate versioned API behavior of onprem Kibana for FTR tests
+            '--server.versioned.versionResolution=oldest',
```

Comment:
We're using the same script to launch Kibana for serverless test configs, so I think this should rather be something like this:
```suggestion
            // Simulate versioned API behavior of Kibana for FTR tests
            config.get('serverless')
              ? '--server.versioned.versionResolution=newest'
              : '--server.versioned.versionResolution=oldest',
```
Or did I miss anything here?

Reply:
Great point!